source/agora/consensus/data/TransactionData.d

```diff
@@ -0,0 +1,297 @@
+/*******************************************************************************
+
+    Contains a type for storing data in a transaction
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.consensus.data.TransactionData;
```

Comment:
How about calling it `DataPayload`? We now have `data.Transaction` and `data.TransactionData` and it's very confusing.

Reply:
Yes, I will fix it